charts/infra-apps/examples/kubernetes-event-exporter.yaml

```diff
@@ -0,0 +1,20 @@
+# Deploy the opsgenie kubernetes-event-exporter
+#
+# This configuration dumps all the events to stdout where they can be picked
+# up by loki.
+
+kubeEventExporter:
+  enabled: true
+  values:
+    config:
+      logLevel: debug
```

Comment:
Why would you want debug as the example level?

Reply:
because it's the default from the upstream chart